schema.graphql

```diff
@@ -71,14 +71,28 @@ type Dispute @entity {
   lastRoundId: BigInt!
   state: DisputeState!
   settledPenalties: Boolean!
-  metadata: String!
+  metadata: Bytes!
+  disputable: Disputable @derivedFrom(field: "dispute")
```

Comment:
Not sure, but is this link actually made?

Reply:
[Here](https://github.com/aragon/court-subgraph/pull/67/files#diff-79e6cdd06d61ff0d7d1f8d8479d453a7R28) :)

Reply:
Ah right, so this will form a bi-directional link?

Reply:
Yes!

Reply:
Yup!